oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -0,0 +1,197 @@
+
+Architectural Advisory Process (AAP)
+------------------------------------
+
+We'll create an Architecture Advisory Process to add structure to today's informal review of ADRs.  The goal of this process is to review ADRs with relevant stakeholders and experts.  Conversations in the advisory process need to take place promptly so that decisions do not feel blocked on gathering advice.
+
+We intend to also use the Advisory Process over time to revisit and iterate on our architectural principles with the goal of ensuring they're specific, measurable, achievable, realistic, and testable. (a variation of SMART criteria)  Similarly, like ADRs and architectural principles, we can use the process to review and adjust Open edX Tech Radar blips.
+
+This process is intended to be lightweight, approachable, and asynchronous.  See :ref:`principles` for more details on the spirit and functioning of the Advisory Process.
+
+Advisory Process Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The final artifact of the Advisory Process is an Architectural Decision Record document checked in to a repository relevant to the decision.  Prior to that, however, we suggest as a best practice using Github's Issues and Pull Request tooling to conduct the Advisory Process.  The advantage of doing so is that Issue/Pull Request comments remove the burden of recording who gave particular advice, and allow us to have threaded, focused conversations about various aspects of ADR.
+
+Note that this section is GitHub-specific because that's where the Open edX platform's code resides.  The idea of "issues" and "pull requests" is otherwise portable to other version control platforms.
+
+GitHub Issues (Optional)
+************************
+
+We would encourage decision makers to use a GitHub Issue to have initial conversations around the context of a decision, or to write down upcoming/potential decisions. It may be that - at the start of a project - we're not even sure what decisions need to be made.  They may not even reveal themselves until we're in the middle of coding.
+
+An Issue is a great place to have these initial, loose, conversations about a decision space without the burden of creating a full ADR (creating a new document in the right place, authoring the sections of an ADR, creating a branch, committing, creating a PR, etc.)  It may not be necessary for more well-defined or focused decisions.  We want to preserve this as an optional first step, as we recognize that creating a "formal" ADR can feel inexplicably daunting or like a distraction, especially when possible decisions come up in the course of working on an implementation.  The mental burden of breaking "flow state" and creating a new ADR document is often problematic.  We hope jotting some notes in an Issue as a starting point will allow engineers to record their potential decision point and quickly continue their work.
```

Comment:
This paragraph is a little overwritten. I think the core point is simple and good and can be very short:

> An Issue is a great place to have these initial, loose conversations about a decision space without the burden of creating a full ADR. We want to preserve this as an optional first step, as we recognize that creating a "formal" ADR can feel daunting or like a distraction. We hope jotting some notes in an Issue as a starting point will allow engineers to record their potential decision point and quickly continue their work.

Reply:
also on the implementation side we would totally want a default button for this sort of issue like we have for DEPR. It doesn't have to have more than the default title and body fields but it should be a first class shiny button

Reply:
Yes!  I totally had that thought and forgot to write it down anywhere.  And yeah, feel free to point out my overwriting, it can happen. 👍🏻

Reply:
I am a chronic overwriter myself 😢

Reply:
Taking @ashultz0's suggestion as-is and blowing away the original paragraph. 😄